themes/bootstrap3/templates/combined/stack-grid.phtml

```diff
@@ -0,0 +1,37 @@
+<div class="combined-search-container grid"
+  style="grid-template-columns: repeat(<?=$this->escapeHtmlAttr($columns)?>, 1fr)"
+>
+  <?php $columnedParams = []; ?>
+  <?php $columnIndex = 0; ?>
+  <?php foreach ($this->combinedResults as $searchClassId => $currentSearch): ?>
+    <?php
+      if (
+        (!isset($currentSearch['ajax']) || !$currentSearch['ajax'])                    // AJAX column
```

Comment:
These checks can be simplified with null coalescing.

Reply:
Addressed by mirroring #3118